app/uk/gov/hmrc/cdsimportsddsfrontend/domain/Commodity.scala

```diff
@@ -16,6 +16,6 @@
 
 package uk.gov.hmrc.cdsimportsddsfrontend.domain
 
-case class Commodity(description: Option[String] = None,
-                     goodsMeasure: Option[GoodsMeasure] = None
-                    )
+case class Commodity(description: Option[String],
+                     classification: Option[Seq[Classification]],
```

Comment:
could this just be a `Seq[_]`, which could be empty, instead of an `Option[Seq[_]]` ?